markdown/developers/create-pipeline.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Pipeline Creation Instructions
+subtitle: Instructions for creating a sanger-tol pipeline
+---
+
+## nf-core instructions
+
+This page is heavily inspired by the nf-core page [Adding a new pipeline](https://nf-co.re/docs/contributing/adding_pipelines).
+
+## Create the pipeline
+
+All pipelines _must_ use the [nf-core template](https://nf-co.re/docs/contributing/guidelines/requirements/use_the_template).
+This is done by using the `nf-core create` command - see [the docs](https://nf-co.re/tools#creating-a-new-pipeline) for detailed instructions.
+This tool does lots of things for you: it gives you the correct file structure and boiler plate code
+and also sets up the required `git` infrastructure for you to keep your pipeline in sync in the future.
+
```

Comment:
Please use `sanger-tol` as the pipeline prefix and you may skip some of template areas.

Reply:
Currently only `iGenomes config` needs to be skipped for some pipelines, not others.

What about the multiqc? no options to skip but we don't want it for some pipelines and it is tedious to remove it later.

Reply:
Very good point. I completely forgot about the configuration options in `nf-core create`. I've added that now.

I don't want to put removing multiqc in the instructions. Yes it's very tedious to remove completely and properly, so I think the easier approach is to simply keep the module around and the various functions and template files, but. not use it in any workflow / sub-workflow. This also simplifies template update as git won't flag those files as merge conflicts.